src/Libraries/RevitNodesUI/Elements.cs

```diff
@@ -43,6 +44,15 @@ protected ElementsQueryBase()
             ShouldDisplayPreviewCore = true;
         }
 
+        [JsonConstructor]
+        public ElementsQueryBase(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPorts) : base(inPorts, outPorts)
+        {
+            var u = RevitServicesUpdater.Instance;
```

Comment:
where was this done before? I just want to make sure this event is unsubscribed from on dispose of this node?

Reply:
Right above is the original non-json constructor doing the same thing.  There is a `Dispose()` method below that when called unsubscribes from the event